dataSources/Trade.csv

```diff
@@ -1,9 +1,65 @@
-TECHNOLOGY,INFUEL,IAR,OUTFUEL,OAR,MODE,CAPACITY (GW),DECOMISSION YEAR,CAPEX,Fixed O&M,Variable O&M,Fuel
-TRNCANMWCANWS,ELCCANMW02,1.05,ELCCANWS01,1,1,5.07,2060,3533,0,0,0
-TRNCANMWCANWS,ELCCANWS02,1.05,ELCCANMW01,1,2,x,x,x,x,x,x
-TRNCANMWCANOT,ELCCANMW02,1.05,ELCCANOT01,1,1,9.46,2060,5037,0,0,0
-TRNCANMWCANOT,ELCCANOT02,1.05,ELCCANMW01,1,2,x,x,x,x,x,x
-TRNCANOTCANQC,ELCCANOT02,1.05,ELCCANQC01,1,1,17.9,2060,6064,0,0,0
-TRNCANOTCANQC,ELCCANQC02,1.05,ELCCANOT01,1,2,x,x,x,x,x,x
-TRNCANATCANQC,ELCCANAT02,1.05,ELCCANQC01,1,1,13.35,2060,4788,0,0,0
-TRNCANATCANQC,ELCCANQC02,1.05,ELCCANAT01,1,2,x,x,x,x,x,x
+TECHNOLOGY,INFUEL,IAR,OUTFUEL,OAR,MODE,CAPACITY (GW),DECOMISSION YEAR,CAPEX,Fixed O&M,Variable O&M,Fuel,Distance,StClairFunc
+TRNCANMWCANWS,ELCCANMW02,1.05,ELCCANWS01,1,1,5.07,2060,3533,0,0,0,x,x
+TRNCANMWCANWS,ELCCANWS02,1.05,ELCCANMW01,1,2,x,x,x,x,x,x,x,x
+TRNCANMWCANOT,ELCCANMW02,1.05,ELCCANOT01,1,1,9.46,2060,5037,0,0,0,x,x
+TRNCANMWCANOT,ELCCANOT02,1.05,ELCCANMW01,1,2,x,x,x,x,x,x,x,x
+TRNCANOTCANQC,ELCCANOT02,1.05,ELCCANQC01,1,1,17.9,2060,6064,0,0,0,x,x
+TRNCANOTCANQC,ELCCANQC02,1.05,ELCCANOT01,1,2,x,x,x,x,x,x,x,x
+TRNCANATCANQC,ELCCANAT02,1.05,ELCCANQC01,1,1,13.35,2060,4788,0,0,0,x,x
+TRNCANATCANQC,ELCCANQC02,1.05,ELCCANAT01,1,2,x,x,x,x,x,x,x,x
+TRNUSAMNUSANW,ELCUSAMN02,1.05,ELCUSANW01,1,1,2.75,2060,2352.941176,0,0,0,600,0.765
+TRNUSAMNUSANW,ELCUSANW02,1.05,ELCUSAMN01,1,2,x,x,x,x,x,x,x,x
+TRNUSACAUSANW,ELCUSACA02,1.05,ELCUSANW01,1,1,6.66,2060,3555.555556,0,0,0,800,0.675
+TRNUSACAUSANW,ELCUSANW02,1.05,ELCUSACA01,1,2,x,x,x,x,x,x,x,x
+TRNUSACAUSAMN,ELCUSACA02,1.05,ELCUSAMN01,1,1,7.11,2060,3555.555556,0,0,0,800,0.675
+TRNUSACAUSAMN,ELCUSAMN02,1.05,ELCUSACA01,1,2,x,x,x,x,x,x,x,x
+TRNUSACAUSASW,ELCUSACA02,1.05,ELCUSASW01,1,1,2.22,2060,3555.555556,0,0,0,800,0.675
+TRNUSACAUSASW,ELCUSASW02,1.05,ELCUSACA01,1,2,x,x,x,x,x,x,x,x
+TRNUSACEUSAMN,ELCUSACE02,1.05,ELCUSAMN01,1,1,0.21,2060,1851.851852,0,0,0,500,0.81

```

Comment:
Maybe there should be a connection between the American Central and Southwest regions, as seen on this map: https://www.eia.gov/energyexplained/electricity/delivery-to-consumers.php, but then again, the geography of American energy seems to vary widely

Reply:
created issue